doc/spec_exceptionHandling.dox

```diff
@@ -122,6 +123,9 @@ A so-called ExceptionHandlingDecorator is placed around all device register acce
 
 \subsection spec_execptionHandling_high_level_implmentation_interface C.1 Internal interface between ExceptionHandlingDecorator and DeviceModule
 
+FIXME: This section is outdated as it does not reflect the introduction of the DeviceManager. In most places it should read "DeviceManager" instead of "DeviceModule", and some of the data members mentioned do not exist any mode.
```

Comment:
Do you want to keep this?

Reply:
Yes, I intentionally put this in so people see that the section is outdated. I also created a redmine ticket.